{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "e4d46a04_222e96a1",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1000331
      },
      "writtenOn": "2022-01-12T17:20:37Z",
      "side": 1,
      "message": "Looks good to me other than resolving whether to log the error 1 or 3 times.  Also needs a rebase.",
      "revId": "6e1c5c9768946e651f256f5225e41347ec46aa63",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}